content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -99,6 +99,237 @@ Printer`. Follow the steps and select the *make* and *model* corresponding to th
+
+   After consulting Epson's website for compatibility for both of the commands: `GS v 0` and
+   `ESC *`, it can be seen that indeed the TM-U220B is not compatible with `GS v 0`, but is
+   compatible with `ESC *`.
+
+   .. image:: troubleshooting/epson-compatibility-compare.png
+      :align: center
+      :alt: Epson compatibility evaluation from Epson website.
+
+   When adding the printer, *CUPS* will ask which printer should be added:
+
+   .. image:: troubleshooting/add-printer.png
+      :align: center
+      :alt: Administration menu, add printer selection.
+
+   In this case, the printer is connected via :abbr:`USB (Universal Serial Bus)` so it won' be part
+   of the :guilabel:`Discovered Network Printers`. Instead it is likely part of the
+   :guilabel:`Unknown` selection. By unplugging the printer's :abbr:`USB (Universal Serial Bus)`
+   cable from the :abbr:`IoT (Internet of Things)` box and refreshing the page, the
+   :guilabel:`Unknown` printer disappears. By plugging it back in, then it reappears, so it can be
+   said that this is the printer in question.
+
+   For the naming convention, since it will need to print using the `ESC *` command, it is
+   imperative to add `__IMC`. With regard to the *density* parameters, toggle to the printer model
+   on `Epson's website
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88>`_.
+
+   .. image:: troubleshooting/epson-TM-U220-specification.png
+      :align: center
+      :alt: Epson TM-U220 specifications on manufacturers website.
+
+   For this particular model, TM-U220, `m` should be equal to 0 or 1. While referencing the
+   :guilabel:`Description` below the pink box in the above picture, the `m` values could be 0, 1, 32
+   or 33. So in this printers case, the `m` value can NOT be 32 or 33 (otherwise there will be
+   random characters printed).
+
+   .. example::
+      If we check the table for the values: 32 and 33, they both occur if the :guilabel:`Number of
+      bits for vertical data` is set to 24. This means that is a *High Vertical Density*. In the
+      case of configuring the Epson TM-U220, the *Low Vertical Density* will need to be forced, as
+      this printer model does not support "High Vertical Density" for this command.
+
+   To add a *Low Vertical Density* add the `LDV` parameter to the naming convention.
+
+   .. image:: troubleshooting/add-printer-filled.png
+      :align: center
+      :alt: Add a *Low Vertical Density* add the `LDV` parameter to the naming convention.
+
+   Click :guilabel:`Continue` to proceed. Next, set the :guilabel:`Make` value to :guilabel:`Raw`
+   and for the :guilabel:`Model` value, set this to :guilabel:`Raw Queue (en)`.
+
+   .. image:: troubleshooting/add-printer-add.png
+      :align: center
+      :alt: Epson TM-U220 specifications on manufacturers website.
+
+   However when trying to print with `EpsonTMU220B__IMC_LDV__` as the naming convention  it will
```

Comment:
the formulation is a bit odd or it is me ? maybe with "as the naming convention is respected/followed" ?

Reply:
Noted, I've reworked this section a bit.